serving/docs/lmi_new/user_guides/lmi-dist_user_guide.md

```diff
@@ -2,8 +2,79 @@
 
 ## Supported Model Architectures
 
+The model architecture that we test for lmi-dist (in CI):
+- Llama (un-quantized and with GPTQ)
+- Falcon
+- GPT-NeoX (un-quantized and with BitsAndBytes)
+- MPT
+- Mistral (un-quantized and with AWQ)
+- Mixtral
+- T5
+
+### Complete Model Set
+
+Optimized models
+* falcon
+* gpt-neox
+* llama
+* llama-2
+* llava
+* mistral
+* mixtral
+* mpt
+* santacoder
+* t5
+
+*Note: PEFT is also supported for optimized models*
+
+`lmi-dist` can also run other models which are supported by huggingface transformers but not listed above, although **without** the optimizations and `gptq`, `awq` quantization support. `transformers` library support can be found for [CausalLM](https://github.com/huggingface/transformers/blob/v4.34.0/src/transformers/models/auto/modeling_auto.py#L381) and [Seq2Seq](https://github.com/huggingface/transformers/blob/v4.34.0/src/transformers/models/auto/modeling_auto.py#L652) models.
+
+
+### Quantization
+
+Currently, we allow customer to use `option.quantize=<quantization-type>` or `OPTION_QUANTIZE=<quantization-type>` to load a quantized model in `lmi-dist`.
+
+We support the following `<quantization-type>`
+* bitsandbytes (>= 0.24.0)
+* bitsandbytes8 (>= 0.25.0)
+* gptq (>= 0.24.0)
+* awq (>= 0.26.0)
```

Comment:
Can we state at the top that the versions refer to LMI container versions (could be confused for other library/package versions)